homework/index.js

```diff
@@ -29,20 +29,23 @@
     return elem;
   }
 
-  function addRepoDetailsInfo(listElm, title, value, link) {
+  function addRepoDetailsInfo(listElm, title, value) {
     const trElm = createAndAppend('tr', listElm);
     createAndAppend('td', trElm, {
       class: 'titles',
       text: title,
     });
-    const secondTdElm = createAndAppend('td', trElm, {
-      class: 'values',
-    });
 
-    if (link) {
-      createAndAppend('a', secondTdElm, { href: link, text: value });
+    if (value === null) {
+      createAndAppend('td', trElm, {
+        class: 'values',
+        text: 'No Information',
+      });
     } else {
-      secondTdElm.innerText = value;
+      createAndAppend('td', trElm, {
+        class: 'values',
+        text: value,
+      });
     }
```

Comment:
Instead of an `if` statement you can use a ternary expression to further simplify your code:

```js
function addRepoDetailsInfo(listElm, title, value) {
  const trElm = createAndAppend('tr', listElm);
  createAndAppend('td', trElm, {
    class: 'titles',
    text: title,
  });
  createAndAppend('td', trElm, {
    class: 'values',
    text: value !== null ? value : 'No Information',
  });
  return trElm;
}
```

Reply:
Thank you so much @remarcmij . I also made last changes.